javascript/leap.js

```diff
@@ -0,0 +1,8 @@
+module.exports = isLeapYear;
+
+function isLeapYear (year) {
+  return (
+    (year % 400 === 0) ||
+    (year % 4 === 0 && year % 100 !== 0)
+  ) ? true : false;
```

Comment:
You could potentially shorten this and eliminate the `if/else` conditionals by returning the result of your logical expressions themselves.